openadmet/toolkit/database/chembl.py

```diff
@@ -545,3 +545,191 @@ def get_variant_ids_for_target(
         group by (variant_id, tid, description, target_chembl_id, assay_id, assay_chembl_id) \
         """
         return self._chembl_connector.query(query, return_as=return_as)
+
+class SemiQuantChEMBLTargetCurator(ChEMBLTargetCuratorBase):
+    """
+    Implement all ChEMBL curation for a given protein target.
+
+    This curation includes both quantitative and semiquantitative data, i.e. activity data that is out of range of the assay.
+    In terms of ChEMBL curation, this means that 'standard_type' can be '=', '<', '>', '<=', or '>='.
+    pChEMBL value is explicitly NOT required.
+
+    """
+
+    standard_type: Optional[str] = Field(
```

Comment:
We don't enforce `standard_type` matching. Instead we match against all on L632. We can add this as an if statement somewhere further down. 

Something like 

```python
if self.standard_type:
    query += f"and activities.standard_type IN ({standard_type})"
else:
    query += "and activities.standard_type IN ('IC50', 'XC50', 'EC50', 'AC50', 'Ki', 'Kd', 'Potency')"